mozilla-services/consvc-shepherd
# [SHEP-114] Added deal name column to the campaigns page view in React UI

## References

[SHEP-114](https://mozilla-hub.atlassian.net/browse/SHEP-114)

## Problem Statement
We need to display the deal names in the campaigns view in the React UI to provide better context regarding which campaign corresponds to each deal.

## Proposed Changes

1. Added the deal name column to the campaigns page in the React UI.
2. Rename 'Net Spend' to 'Budget.

## Verification Steps

1. Run the React front end locally.
2. Verify that the deal name column is added to the campaigns page.

## Visuals
<img width="1405" alt="Screenshot 2024-10-30 at 1 19 34 PM" src="https://github.com/user-attachments/assets/0fb09d2d-3e94-4d79-a8ba-6528d23a4bbc">



[SHEP-114]: https://mozilla-hub.atlassian.net/browse/SHEP-114?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
